roles/os_net_setup/defaults/main.yml

```diff
@@ -6,15 +6,17 @@ cifmw_os_net_setup_osp_calls_delay: 5
 cifmw_os_net_setup_config:
   - name: public
     external: true
+    shared: true
+    mtu: 1330
```

Comment:
why mtu reduction?

Apart from that all looks good.

Reply:
This is for whitebox path mtu discovery tests (the ones under test_mtu.py). For this coverage our tripleo jobs had external network mtu 1500 and tenant mtu 9000. When we were planning osp18 jobs I suggested for simplification to use normal tenant networks mtu but just reduce external network mtu. So this is it.
Maybe we should have set this only for neutron jobs, not for everybody.

Reply:
+1 to only override where it's really needed, not in default

Reply:
I removed mtu reduction